masonry/src/widget/sized_box.rs

```diff
@@ -266,6 +275,18 @@ impl SizedBox {
         this.ctx.request_paint_only();
     }
 
+    /// Clears padding.
+    pub fn clear_padding(this: &mut WidgetMut<'_, Self>) {
+        this.widget.padding = Insets::ZERO;
+        this.ctx.request_paint_only();
+    }
+
+    /// Set the padding around this widget.
+    pub fn set_padding(this: &mut WidgetMut<'_, Self>, padding: impl Into<Insets>) {
+        this.widget.padding = padding.into();
+        this.ctx.request_paint_only();
```

Comment:
This is surprising to me. How does this work? The padding field is used in `layout`, so I'd expect changing it to have an impact on layout.

Could we have a test which changes the padding at runtime and demonstrates that the correct result is reached?

Reply:
This is a mistake, it should request layout change.

Reply:
It's fixed now